CONTRIBUTING.md

```diff
@@ -184,9 +184,9 @@ New Relic hosts and moderates an online forum where customers can interact with
 New Relic employees as well as other customers to get help and share best
 practices. Like all official New Relic open source projects, there's a related
 Community topic in the New Relic Explorers Hub. You can find this project's
-topic/threads here:
+topic/threads under the "Ruby Agent" category here:
 
-[Explorer's Hub](https://discuss.newrelic.com/tags/rubyagent)
+[Explorer's Hub](https://forum.newrelic.com/s/)
```

Comment:
This gets a little closer to the category:
```suggestion
[Explorer's Hub](https://forum.newrelic.com/s/category/Category__c/Default)
```

Reply:
perfect!